### PR TITLE
Add login command

### PR DESCRIPTION
**Pull request by montezdesousa**
# Description

* set in .env `OPENBB_ENABLE_AUTHENTICATION='True'` to test
* Also adds enter for guest mode

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by jmaslek** [comment]
I must have previously used the 'remember me' option, but login when already logged in throws an error:

```
2023 Mar 17, 10:02 (🦋) /account/ $ login

You are already logged in.



The command 'login' doesn't exist on the / menu.
```

(if it matters, I overwrote the is_auth_enabled() to always be true)

**Comment by jmaslek** [comment]
Seems even logging out then in:
```
2023 Mar 17, 10:04 (🦋) /account/ $ logout

You are currently logged as a guest.
Register: https://my.openbb.dev/register


2023 Mar 17, 10:04 (🦋) /account/ $ login

You are already logged in.



The command 'login' doesn't exist on the / menu.
```

**Comment by montezdesousa** [comment]
> I must have previously used the 'remember me' option, but login when already logged in throws an error:
> 
> ```
> 2023 Mar 17, 10:02 (🦋) /account/ $ login
> 
> You are already logged in.
> 
> 
> 
> The command 'login' doesn't exist on the / menu.
> ```
> 
> (if it matters, I overwrote the is_auth_enabled() to always be true)

yesss, forgot a `not` there

**Review by jmaslek — APPROVED** [review]

File: terminal.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def main():
     else:
         from openbb_terminal.core.session import session_controller
 
-        if is_auth_enabled():
+        if is_auth_enabled() and ("--login" in sys.argv[1:] or is_installer()):
```
Comment (hjoaquim) [inline terminal.py:22]: Will installer users need to set the flag in the `.env` to test the login functionality?

**Comment by hjoaquim** [comment]
This PR also fixes #4302. Well done @montezdesousa !

File: terminal.py (hunk 1 above)
Comment (jmaslek) [reply terminal.py:22]: i just make this function return True lol.  But until we launch, yeah

**Pull request merged**